modules/friendlytag.js

```diff
@@ -183,6 +183,47 @@ Twinkle.tag.callback = function friendlytagCallback() {
 		case 'redirect':
 			Window.setTitle('Redirect tagging');
 
+			// https://stackoverflow.com/questions/280634/endswith-in-javascript/2548133#2548133
+			// Author chakrit, CC BY-SA 4.0
+			var endsWith = function(str, suffix) {
+				return str.indexOf(suffix, str.length - suffix.length) !== -1;
```

Comment:
Not necessary to implement this anymore - we can use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith which is possible now as IE 11 is no longer supported.